src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsauditapi/resource/AuditResource.kt

```diff
@@ -67,21 +77,35 @@ class AuditResource(
   }
 
   @PreAuthorize("hasRole('ROLE_AUDIT') and hasAuthority('SCOPE_read')")
-  @GetMapping("/paged")
+  @PostMapping("/paged")
```

Comment:
qq: why was this changed to POST and a DTO? was there an issue with new parameters being added to the GET?

Reply:
It was changed to a DTO to introduced validation and data binding logic in a cleaner way outside the controller method. Also, using POST gives us more flexibility with posting a json, which is good practice as params list grows. We don't want a long list of params as method arguments, that begs for using a POJO/JSON.